force-app/main/releaseGating/lwc/releaseGateItem/releaseGateItem.html

```diff
@@ -0,0 +1,172 @@
+<template>
+    <section>
+        <ol class="slds-setup-assistant slds-theme_default">
+            <li class="slds-setup-assistant__item">
+                <article class="slds-setup-assistant__step">
+                    <div class="slds-summary-detail slds-is-open">
+                        <div class="slds-container_fluid">
+                            <div class="slds-summary-detail__title">
+                                <div class="slds-setup-assistant__step-summary">
+                                    <div class="slds-media">
+                                        <div class="slds-media__figure gateIcon">
+                                            <lightning-icon
+                                                icon-name="action:announcement"
+                                                if:true={gateDisabledOrInactive}
+                                                size="x-small"
+                                            >
+                                            </lightning-icon>
+                                            <lightning-icon
+                                                icon-name="action:approval"
+                                                if:true={gateActive}
+                                                size="x-small"
+                                            >
+                                            </lightning-icon>
+                                            <lightning-icon
+                                                icon-name="action:more"
+                                                if:true={gateInProgress}
+                                                size="x-small"
+                                            >
+                                            </lightning-icon>
+                                        </div>
+                                        <div class="slds-media__body slds-m-top_x-small">
```

Comment:
Please convert your margin and variable classes to the new slds-var-* format
```suggestion
                                        <div class="slds-media__body slds-var-m-top_x-small">
```

Reply:
Fixed outdated css classes